.circleci/config.yml

```diff
@@ -1224,22 +1260,36 @@ jobs:
           name: Prepare test environment (compile tests and cache results)
           working_directory: op-acceptance-tests
           command: go test -v ./... || true # ignore failures; we're just compiling the tests
-      # Run the actual acceptance tests with optimized settings
       - run:
-          name: Run acceptance tests (devnet=<<parameters.devnet>>, gate=<<parameters.gate>>)
+          name: Stop the job if the devnet failed to start
+          command: circleci-agent step halt
+          when: on_fail
+      # Run the acceptance tests (if the devnet is running)
+      - run:
+          name: Run acceptance tests (gate=<<parameters.gate>>)
           working_directory: op-acceptance-tests
-          no_output_timeout: 2h
+          no_output_timeout: 1h
           environment:
             GOFLAGS: "-mod=mod"
             GO111MODULE: "on"
-          command: just acceptance-test "<<parameters.devnet>>" "<<parameters.gate>>"
+          command: |
+            # Mark that we're in the test step
+            echo 'export RUNNING_TESTS="true"' >> $BASH_ENV
```

Comment:
What is this variable used for?

Reply:
Ah, nothing. I was experimenting and forgot to remove this. Thanks - good catch!